editing.py

```diff
@@ -217,9 +254,37 @@ def add_artist(self, artist, edit_note, auto=False):
         self.b.submit()
         return self._extract_mbid("artist")
 
-    def add_area(self, area, edit_note, auto=False):
-        # Unlike other methods, this one directly requests page using
-        # mechanize.Requests to overcome mechanize's lack of javascript support
+    def add_area(self, area: dict, edit_note: str, auto=False) -> str:
+        """A method to add a new area to MusicBrainz
+
+        Args:
+            area (dict): a dictionary containing the area's data (format below)
+            edit_note (str): edit note
+            auto (bool, optional): Unused, kept to maintain format. Defaults to False.
```

Comment:
auto marks whether it should be entered as a votable edit or as an auto-edit. I probably would expect these to be auto-edits, but we'd need to decide.

Reply:
Thanks, I'll update it in the PR :)